config/data_sync_list/open-power.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Directories": [
+        {
+            "Path": "/var/lib/phosphor-logging/errors",
```

Comment:
There's going to need to be code changes in phosphor-logging before any syncing can be done, such as creating the filenames/log IDs based on the BMC position.  Otherwise, the errors will just overwrite the other one.

Reply:
ok. I can remove this one as code changes is not there. Will add some other file details .Added some of the files/directories into each JSON files to enable the application framework.